docs/installation.rst

```diff
@@ -301,6 +301,20 @@ Install the dependencies with development headers::
     sudo dnf install redhat-rpm-config                 # not needed in Korora
     sudo dnf install fuse-devel fuse pkgconfig         # optional, for FUSE support
 
+openSUSE Tumbleweed / Leap
+++++++++++++++++++++++++++
+
+Install the dependencies automatically using zypper::
+
+    sudo zypper source-install --build-deps-only borgmatic
```

Comment:
Oops sorry, I've been working also with borgmatic lately and wrote it there by mistake. I just fixed it